cuqipy_fenics/pde.py

```diff
@@ -18,10 +19,23 @@ class FEniCSPDE(PDE,ABC):
 
     Parameters
     ----------
-    PDE_form : python function handle
-        Function handle of a python function that returns the FEniCS weak form of the PDE.
-
-        This python function takes as input, in this given order, the Bayesian parameter (input of the forward model), the state variable (solution variable), the adjoint variable (which is also the test variable in the weak formulation) as FEniCS functions (or FEniCS trail or test functions). See, for example, `demos/demo03_poisson_circular.py` for an example of how to define this function.
+    PDE_form : python function handle or a tuple of two python function handles
```

Comment:
Good suggestion. I updated this line. I ended up using callable (similar to [spicy](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.minimize_scalar.html#scipy.optimize.minimize_scalar) documentation for example) instead of Function because Function is a commonly used object type in FEniCS.

Reply:
Great! Callable is the right terminology.